apps/docs/docs/templates/overview.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+## Purpose
+
+To provide pre-built integration workflows that are commonly used to accelerate the process of building integration applications
+
+designed to streamline the development by providing a baseline or a starting point with predefined business logic and configurations
```

Comment:
NIT: Uppercase `D`?

Reply:
haha totally missed that part, thanks - changed to point form